app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError("Command must be in the "
+                         "format 'cp <file.txt> <file-copy.txt>'")
+    source_file, destanation_file = parts[1], parts[2]
```

Comment:
There is a typo in the variable name `destanation_file`. It should be `destination_file`.